ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__nutanix-4.18-nightly-x86.yaml

```diff
@@ -51,13 +52,15 @@ tests:
     allow_skip_on_success: true
     cluster_profile: nutanix-qe
     env:
-      COMPUTE_REPLICAS: "24"
+      COMPUTE_REPLICAS: "3"
       ITERATIONS_PER_NAMESPACE: "400"
       PROFILE_TYPE: reporting
+      WORKER_REPLICA_COUNT: "24"
     test:
+    - chain: workers-scale
     - chain: openshift-qe-node-density-heavy
     workflow: openshift-qe-installer-nutanix
-  timeout: 6h0m0s
```

Comment:
do you need this timeout to be increased? can we leave at 6 hours for merging?

Reply:
Thanks for the suggestion Paige. I think its better if we do it 8 hrs